modules/moving-the-gitops-operator-pod-to-infrastructure-nodes.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assembly:
+//
+// * gitops_workloads_infranodes/run-gitops-control-plane-workload-on-infra-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="moving-the-gitops-operator-pod-to-infrastructure-nodes_{context}"]
+= Moving the {gitops-shortname} Operator pod to infrastructure nodes
+
+You can move the {gitops-shortname} Operator pod to the infrastructure nodes.
+
+.Prerequisites
+* {gitops-title} is installed in your cluster.
+
+.Procedure
+
+. Label existing nodes as infrastructure by running the following command:
++
+[source,terminal]
+----
+$ oc label node <node-name> node-role.kubernetes.io/infra=
+----
++
+.Example output
+[source,terminal]
+----
+node/<node-name> labeled
+----
+
+. Edit the {gitops-title} `Subscription` resource to specify where the Operator should run:
++
+[source,terminal]
+----
+$ oc -n openshift-gitops-operator edit subscription openshift-gitops-operator
+----
+
+. Add `nodeSelector` and `tolerations` to the `spec.config` field in the `Subscription` resource:
++
+.Example Subscription
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-gitops-operator
+  namespace: openshift-gitops-operator
+spec:
+  config:
+    nodeSelector:
+      node-role.kubernetes.io/infra: ""
+    tolerations:
+    - key: node-role.kubernetes.io/infra
+      operator: Exists
+      effect: NoSchedule
+----
++
+.Example output
+[source,terminal]
+----
+subscription.operators.coreos.com/openshift-gitops-operator edited
+----
+
+
```

Comment:
@reginapizza can we run any verification steps similar to [this procedure](https://docs.openshift.com/container-platform/4.13/service_mesh/v2x/installing-ossm.html#ossm-confirm-operator-infrastructure-node_installing-ossm)?

Reply:
Good question @ekristova.  Can you please try this? What results are you getting? 

I suggest you document it here and @reginapizza can have a look and provide feedback.

Reply:
Hello @reginapizza I added the verification step, but I am not sure what should the user look for in there. Can you please check the step and output that I provided and let me know what we should look for? Thank you!

Reply:
So afaik, the way you tell the pod is running on an infra node is by looking at the pod and checking what node it's running on, and then looking at the node to make sure it has the `node-role.kubernetes.io/infra: ""` label applied (which you did already in a previous step when you labeled it).

Reply:
@reginapizza Oh I see, thank you for the explanation! The wording here made it seem as if we could see the label from that command. So I will rephrase the verification step to: 
`Verify that the {gitops-shortname} Operator pod is running on the infrastructure node :`
That clarifies what should the user look for. Thank you!